blog/features/dfx-deploy-playground.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Introducing `dfx deploy --playground`!
+description: Today we're excited to announce a new dfx subcommand — `dfx deploy --playground`!
+tags: [New features]
+image: /img/blog/deploy-playground.jpg
+---
+
+# Introducing `dfx deploy --playground`
+
+Today we're excited to announce a new `dfx` subcommand: `dfx deploy --playground`!
+
+`--playground` (or `--network playground`) is a new network flag for the `dfx deploy` command that deploys the project's canister(s) to the Motoko playground's canister pool instead of deploying them to the local replica (`--network local`) or the mainnet network (`--network ic`). 
+
+When this flag is used, the canisters are deployed in a manner where they borrow resources from the Motoko playground's canister pool. The Motoko playground provides developers with an environment where canisters can be deployed and tested quickly without needing to configure a cycles wallet. 
+
+`dfx deploy --playground` is available in `dfx` versions v0.15.1 and above.
+
+## What this means for developers
+
+Since the `dfx deploy --playground` feature doesn't require additional configuration, developers can get up and running with very little prerequisites, making the on-boarding of new developers more seamless than before. For existing developers, this provides a new way for canisters to be tested, since canisters deployed to the playground are deployed on the mainnet, but do not cost cycles that the developer must pay. This means developers can run short tests using mainnet resources, without actually having to set up a well-funded and permanent identity. 
```

Comment:
```suggestion
Since the `dfx deploy --playground` feature doesn't require additional configuration, developers can get up and running with very few prerequisites, making the on-boarding of new developers more seamless than before. For existing developers, this provides a new way for canisters to be tested, since canisters deployed to the playground are deployed on the mainnet, but do not cost cycles that the developer must pay. This means developers can run short tests using mainnet resources, without actually having to set up a well-funded and permanent identity. 
```